mdm/orientdb-wrapper/src/main/java/com/gentics/mesh/core/data/dao/impl/SchemaDaoWrapperImpl.java

```diff
@@ -146,7 +146,7 @@ public Result<? extends HibNodeFieldContainer> findDraftFieldContainers(HibSchem
 
 	@Override
 	public Result<HibProject> findLinkedProjects(HibSchema schema) {
-		return new TraversalResult<>(getRoots(schema).stream().map(root -> root.getProject()));
+		return new TraversalResult<>(getRoots(schema).stream().map(root -> root.getProject()).filter(Objects::nonNull));
```

Comment:
In the test AW MP database there was a case when a ODB edge existed, but lead to nothing.